.github/workflows/publish.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Publish Packages
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish:
+    if: startsWith(github.event.head_commit.message, 'Publish') == true
```

Comment:
QQ: What would happen if we don't give perform this if condition?

Reply:
Then every-time , it would be merged to master , Will publish a given version .
If we wanna publish only when we want it to be published , then we need this If condition.